src/clients/servicebus/ServiceBusClient.js

```diff
@@ -5,21 +5,25 @@ const azure = require('azure-sb');
 const trackDependency = require('../appinsights/AppInsightsClient').trackDependency;
 const SERVICE_BUS_CONNECTION_STRING = process.env.FORTIS_SB_CONN_STR;
 
-let client = azure.createServiceBusService(SERVICE_BUS_CONNECTION_STRING);
-
 /**
  * @param {string} queue
  * @param {string} message
  * @returns {Promise}
  */
 function sendStringMessage(queue, message) {
   return new Promise((resolve, reject) => {
+    let client;
+    try {
+      client = azure.createServiceBusService(SERVICE_BUS_CONNECTION_STRING);
+      if (!client) return reject('Failed to create service bus service. No service bus connection string provided.');
```

Comment:
Nit: This can live outside of the try block so that you don't risk catching spurious exceptions.